.github/workflows/windows-ci.yml

```diff
@@ -0,0 +1,60 @@
+name: windows-ci
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+env:
+  rust-version: 1.62.0
+  python-version: "3.9"
+  poetry-version: 1.1.13
+  sqlite-version: 3390100
+  winfsp-version: 1.8.20304
+
+jobs:
+  python-only:
+    runs-on: windows-2019
```

Comment:
Why 2019 ? Why not the latest windows?

Reply:
I've got some problems with `windows-2022` on the rust part when installing `diesel-rs`, but I could try to use it for the python part